CAIPs/caip-261.md

```diff
@@ -0,0 +1,466 @@
+"issuer": "did:pkh:eip155:1:0x44dc4E3309B80eF7aBf41C7D0a68F0337a88F044",
+"credentialStatus": {
+    "id": "ipfs://QmcwYEnWysTyepjjtJw19oTDwuiopbCDbEcCuprCBiL7gt",
+    "type": "CredentialStatus",
+    "statusPurpose": "revocation",
+},
+```
+##### On-chain assertions revocation
+Most on-chain assertion storages manage revocation natively without requiring changes to the overall schema.
+
+#### Trust Graph Expansion
+
+##### Implicit Trust Assertions
+
+While the Peer Trust Assertion model outlined previously focuses on explicit trust signals, the trust graph can be further enriched with additional implicit on-chain and/or off-chain trust signals related to its peers. These can include Proof of Humanity, Proof of Membership, Proof of Contributions, Proof of Attendance, Social Graphs, among others.
+This specification does not define the handling of implicit trust; it leaves the integration of relevant data to the discretion of the final consumer, depending on the requirements of the specific use-case.
+
+##### Additional Explicit Trust Assertions
+
+The trust graph can also be enhanced with additional explicit trust signals derived from any model, broadening the scope of trust assessment.
+
+### Trust Assertion Verification
+
+The standard presumes that both the `issuer` property will be dereferenced and the complete contents of the `credentialSubject` will be consumed only after the wire-formats and signed-envelopes have been verified.
+
+#### Signature Verification
+
+The veracity and integrity of trust assertions are paramount to ensuring unbiased insights.
+All trust assertions SHOULD be cryptographically signed by the issuer using strong cryptographic methods and verified prior to consumption.
+The specification supports any strong signature methods, such as: ECDSA, EdDSA, Schnorr Signatures, RSA, etc.
+
+[EIP-712][] is recommended to implementers as a complementary cryptographic proof method alongside others like ECDSA, EdDSA, Schnorr, and RSA for on-chain verifiable credentials.
+Its inclusion emphasizes the importance of user-friendly, secure, and efficient interactions with blockchain-based identity and credential systems.
+It is noteworthy that [EIP-712][] mandates the presence of all fields, even if some are left empty in order to enable their verification.
+Note that composing [EIP712][] with [W3C Verifiable Credentials][VC] requires the [EthereumEip712Signature2021][] signature suite.
+
+##### Example EIP-712 proof object for use in a W3C Verifiable Credential
+
+```json
+  "proof": {
+      "verificationMethod": "did:pkh:eip155:59144:0x3892967AA898d7EeBf1B08d3E1F31B2F4C84317A#blockchainAccountId",
+      "created": "2024-01-27T07:05:56.273Z",
+      "proofPurpose": "assertionMethod",
+      "type": "EthereumEip712Signature2021"
+  }
+```
+##### On-chain assertions verification
+On-chain the verification is automaticaly done by the trusted protocol, but can be verified according to the protocol specificities.

```

Comment:
```suggestion
When computed "on-chain" (i.e. in a node runtime), the verification logic is automaticaly handled by core transaction logic, but can also be verified independently by mirroring this logic elsewhere or after the fact from on-chain records.
```
I'm not sure if this was exactly what you meant, but if not, please be more explicit about whatever that was, it's cryptic in its current form :D

Reply:
That's correct. I just wanted to say that trusting the blockchain implementation and immutability make the computation trustless without the need of verification at each compute